tests/tests.schema

```diff
@@ -665,7 +665,239 @@
           "description": "Schema to fill in.",
           "anyOf": [
             {
-              "$ref": "#/definitions/schema"
+              "$schema": "http://json-schema.org/draft-07/schema#",
+              "title": "Core schema meta-schema",
+              "definitions": {
+                "schemaArray": {
+                  "type": "array",
+                  "minItems": 1,
```

Comment:
If you are going to regenerate this file you need to make sure you are running the latest bf dialog:merge by doing bf plugins:install @microsoft/bf-dialog@next